cmd/mc-client-test/lcdemo.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+
+# Copyright 2023 The KubeStellar Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+. demo-magic.sh
+clear
+
+TYPE_SPEED=20
+#DEMO_PROMPT="\[\033[01;32m\]\u@\h\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ "
+DEMO_PROMPT="\[\033[01;32m\]\u@WSL2\[\033[00m\]:\[\033[01;34m\]\[\033[00m\]\$"
+
+MGT_NAME="mgt"
+MGT_CTX="kind-$MGT_NAME"
+
+# For live demo use 'pe'  for recording or auto demo use 'pei'
```

Comment:
Not a blocker: a more evolved script would take a parameter indicating which mode to operate in, rather than require editing the script.

Reply:
We are in the process of changing mc-client-test to be more generic (as well as create a more general test script).  This will be pushed via a dedicated PR.